spec/index.bs

```diff
@@ -2253,6 +2253,7 @@ handling should allow it to recover.
 
 Any {{GPUAdapter}} returned by {{GPU/requestAdapter()}} must provide the following guarantees:
 
+- At least one of the following must be true:
```

Comment:
something happened here

Reply:
I think that happened after I did commit suggestion for this: https://github.com/gpuweb/gpuweb/pull/4763#discussion_r1690550313